pandas/tests/arrays/boolean/test_arithmetic.py

```diff
@@ -13,30 +15,87 @@ def data():
     )
 
 
-class TestArithmeticOps(BaseOpsUtil):
-    def test_error(self, data, all_arithmetic_operators):
-        # invalid ops
+@pytest.fixture
+def a():
+    return pd.array([True] * 3 + [False] * 3 + [None] * 3, dtype="boolean")
 
-        op = all_arithmetic_operators
-        s = pd.Series(data)
-        ops = getattr(s, op)
-        opa = getattr(data, op)
 
-        # invalid scalars
-        with pytest.raises(TypeError):
-            ops("foo")
-        with pytest.raises(TypeError):
-            ops(pd.Timestamp("20180101"))
+@pytest.fixture
+def b():
```

Comment:
would it be burdensome to give a and b more informative names?

Reply:
Do you have a suggestion what you would find clearer?

Basically, before moving it to a fixture, I had:

```
def test_..():
    a = pd.array([True] * 3 + [False] * 3 + [None] * 3, dtype="boolean")
    b = pd.array([True, False, None] * 3, dtype="boolean")

    result = op(a, b)
```

in each test, and I would think in such a context you would be fine with those names? 
And although I like defining those variables inside the test as more explicit, it was also a bit repetitive, so I moved it to a fixture.

Reply:
i would call these left_array, right_array

Reply:
Renamed